Sources/Application/main.swift

```diff
@@ -1 +1,12 @@
-print("Hello")
\ No newline at end of file
+import SnowManLogic
+import Dictionary
+
+let dictionaryManager = DictionaryManager(words: words) 
```

Comment:
I tried to do that earlier but because Swift has a `Dictionary` already, I just changed all of my references to the `Dictionary` package to `SnowManDictionary`